src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -436,12 +436,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                     ->info('validation configuration')
                     ->canBeEnabled()
                     ->children()
-                        ->scalarNode('cache')
-                            ->beforeNormalization()
-                                // Can be removed in 3.0, once ApcCache support is dropped
-                                ->ifString()->then(function ($v) { return 'apc' === $v ? 'validator.mapping.cache.apc' : $v; })
```

Comment:
Three options then:
- trigger a deprecation in 2.8, now released ? (as a "bugfix": a missing deprecation notice)
- keep the ability to pass "apc", but trigger a new deprecation in 3.0. Then, remove this ability in 3.1
- keep the ability to pass "apc"